src/pages/StockPage.js

```diff
@@ -48,10 +48,13 @@ export const Stock = ({
   onDeselectRow,
   onFilterData,
   onSortColumn,
+  refund,
 }) => {
   //  Refresh data on retrieving item or itembatch records from sync.
   useSyncListener(refreshData, ['Item', 'ItemBatch']);
 
+  const refundCallback = React.useCallback(() => itemId => refund(itemId), []);
```

Comment:
Nit: `React.useCallback()` can just be `useCallback()`.

Reply:
I've been using it from `React` so I don't have to import and I like it more changemymind